src/main/java/cn/nukkit/block/BlockEndRod.java

```diff
@@ -0,0 +1,75 @@
+package cn.nukkit.block;
+
+import cn.nukkit.Player;
+import cn.nukkit.item.Item;
+import cn.nukkit.item.ItemTool;
+import cn.nukkit.math.AxisAlignedBB;
+import cn.nukkit.math.BlockFace;
+
+/**
+ *
+ * @author PikyCZ
+ */
+public class BlockEndRod extends BlockTransparent {
+
+    public BlockEndRod() {
+        this(0);
+    }
+
+    public BlockEndRod(int meta) {
+        super(meta);
+        System.out.println(meta + "");
+    }
+
+    @Override
+    public String getName() {
+        return "End Rod";
+    }
+
+    @Override
+    public int getId() {
+        return END_ROD;
+    }
+
+    @Override
+    public double getHardness() {
+        return 2;
+    }
+
+    @Override
+    public double getResistance() {
+        return 15;

```

Comment:
http://minecraft.gamepedia.com/End_Rod @PikyCZ 
WHO TOLD YOU THAT THE  RESISTANCE IS 15? IT IS 0 INSTEAD!!!